src/ui/actions/timeline.ts

```diff
@@ -360,17 +366,8 @@ export function setHoveredPoint(hoveredPoint: HoveredPoint | null): UIThunkActio
   return ({ dispatch, getState }) => {
     dispatch({ type: "set_hovered_point", hoveredPoint });
 
-    dispatch(setTimelineTooltip(hoveredPoint ? hoveredPoint.time : null));
-
     // Don't update the video if user is adding a new comment.
-    if (selectors.getPendingComment(getState())) {
-      return;
-    }
-
-    if (hoveredPoint) {
-      paintGraphicsAtTime(hoveredPoint.time);
-    } else {
-      paintGraphicsAtTime(selectors.getCurrentTime(getState()));
-    }
+    const updateGraphics = !selectors.getPendingComment(getState());
+    dispatch(setTimelineToTime(hoveredPoint?.time || null, updateGraphics));
```

Comment:
when would hoveredPoint not have a time?

Reply:
https://github.com/RecordReplay/devtools/blob/3d8fb302472837ed29aa92b96193c2b28684a4d8/src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.js#L101-L106